build_projects/shared-build-targets-utils/Utils/Dirs.cs

```diff
@@ -8,38 +8,11 @@ namespace Microsoft.DotNet.Cli.Build
     public static class Dirs
     {
         public static readonly string RepoRoot = Directory.GetCurrentDirectory();
-
-        public static readonly string DebPackagingConfig = Path.Combine(Dirs.RepoRoot, "packaging", "deb");
         public static readonly string Output = Path.Combine(
```

Comment:
Next PR. The Publish/Finalize targets are pretty involved and will warrant their own cleanup PR.